library/management/command/src/main/java/org/quiltmc/qsl/command/api/EntitySelectorOptionRegistry.java

```diff
@@ -29,10 +29,10 @@
 
 /**
  * Class to allow registration of custom {@link net.minecraft.command.EntitySelectorOptions entity selector options}.
- *
- * <p>These are registered with namespaced IDs to avoid name collisions. These are then converted to a name of the form
+ * <p>
+ * These are registered with namespaced IDs to avoid name collisions. These are then converted to a name of the form
  * "namespace_path". Due to {@link com.mojang.brigadier.StringReader#isAllowedInUnquotedString(char) limitations} in how
```

Comment:
```suggestion
 * "namespace_path". Due to {@linkplain com.mojang.brigadier.StringReader#isAllowedInUnquotedString(char) limitations} in how
```
Fixes "limitations" showing up as `limitations`. :P

Reply:
thank you :)